assets/js/modules/analytics/datastore/properties.js

```diff
@@ -0,0 +1,325 @@
+		invariant( property, 'property is required' );
+
+		return {
+			payload: { accountId, property },
+			type: RECEIVE_CREATE_PROPERTY,
+		};
+	},
+
+	/**
+	 * Logs an error with property creation.
+	 *
+	 * @since n.e.x.t
+	 * @private
+	 *
+	 * @param {Object} args Argument params.
+	 * @param {string} args.accountId Google Analytics account ID.
+	 * @param {Object} args.error Error object.
+	 * @return {Object} Redux-style action.
+	 */
+	receiveCreatePropertyFailed( { accountId, error } ) {
+		invariant( accountId, 'accountId is required' );
+		invariant( error, 'error is required.' );
+
+		return {
+			payload: { accountId, error },
+			type: RECEIVE_CREATE_PROPERTY_FAILED,
+		};
+	},
+
+	receivePropertiesProfiles( { accountId, properties, profiles } ) {
+		invariant( accountId, 'accountId is required' );
+		invariant( properties, 'properties is required' );
+
+		return {
+			payload: { accountId, properties, profiles },
+			type: RECEIVE_PROPERTIES_PROFILES,
+		};
+	},
+
+	receivePropertiesProfilesFailed( { accountId, error } ) {
+		invariant( accountId, 'accountId is required' );
+		invariant( error, 'error is required.' );
+
+		return {
+			payload: { accountId, error },
+			type: RECEIVE_PROPERTIES_PROFILES_FAILED,
+		};
+	},
+};
+
+export const controls = {
+	[ FETCH_CREATE_PROPERTY ]: ( accountId ) => {
+		return API.set( 'modules', 'analytics', 'create-property', { accountID: accountId } );
+	},
+	[ FETCH_PROPERTIES_PROFILES ]: ( accountId ) => {
+		return API.get( 'modules', 'analytics', 'properties-profiles', { accountID: accountId } );
+	},
+};
+
+export const reducer = ( state, action ) => {
+	switch ( action.type ) {
+		case FETCH_CREATE_PROPERTY: {
+			const { accountId } = action.payload;
+
+			return {
+				...state,
+				isDoingCreateProperty: {
+					...state.isDoingCreateProperty,
+					[ accountId ]: true,
+				},
+			};
+		}
+
+		case FETCH_PROPERTIES_PROFILES: {
+			const { accountId } = action.payload;
+
+			return {
+				...state,
+				isFetchingPropertiesProfiles: {
+					...state.isFetchingPropertiesProfiles,
+					[ accountId ]: true,
+				},
+			};
+		}
+
+		case RECEIVE_CREATE_PROPERTY: {
+			const { accountId, property } = action.payload;
+
+			return {
+				...state,
+				isDoingCreateProperty: {
+					...state.isDoingCreateProperty,
+					[ accountId ]: false,
+				},
+				properties: [
+					...state.properties || [],
+					property,
+				],
+			};
+		}
+
+		case RECEIVE_CREATE_PROPERTY_FAILED: {
+			const { accountId, error } = action.payload;
+
+			return {
+				...state,
+				error,
+				isDoingCreateProperty: {
+					...state.isDoingCreateProperty,
+					[ accountId ]: false,
+				},
+			};
+		}
+
+		case RECEIVE_PROPERTIES_PROFILES: {
+			const { accountId, properties, profiles } = action.payload;
+
+			return {
+				...state,
+				isFetchingPropertiesProfiles: {
+					...state.isFetchingPropertiesProfiles,
+					[ accountId ]: false,
+				},
+				properties,
+				profiles,
```

Comment:
Same here, profiles should be integrated into a `state.profiles` that is keyed by account ID on the top level and by property ID on the second level.